sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
Okay, you're right. Instead of
> so that the frame is immediately sent if the channel is clear. 

it should be

> so that the channel is sensed immediately without random backoff.